flink-ml-lib/src/main/java/org/apache/flink/ml/feature/vectorassembler/VectorAssembler.java

```diff
@@ -47,10 +47,15 @@
 
 /**
  * A Transformer which combines a given list of input columns into a vector column. Types of input
- * columns must be either vector or numerical value.
+ * columns must be either vector or numerical types. The elements assembled in the same column must
+ * have the same size. If the element is null or has the wrong size, we will process this case with
+ * {@link HasHandleInvalid} parameter as follows:
  *
- * <p>The `keep` option of {@link HasHandleInvalid} means that we output bad rows with output column
- * set to null.
+ * <p>The `keep` option means that we do not check the vector size, and keep all rows.
```

Comment:
If the size does not match the expected, what would be the output value of the row?

Reply:
The output value will have a different vector size with the normal ones.